terraform/modules/policies/alternate_accounts.tf

```diff
@@ -0,0 +1,25 @@
+#resource "aws_account_alternate_contact" "security" {
+#
+#  alternate_contact_type = "SECURITY"
+#
+#  name          = "GDX Dev Team"
+#  title         = "Dev Team"
+#  email_address = "gdx-dev-team@digital.cabinet-office.gov.uk"
+#  phone_number  = sensitive(data.aws_ssm_parameter.security_contact_number.value)
+#}
+
+resource "aws_ssm_parameter" "security_contact_number" {
+  name  = "security-contact-number"
+  type  = "SecureString"
+  value = "+123456789"
+
+  lifecycle {
+    ignore_changes = [
+      value
+    ]
+  }
+}
+
+data "aws_ssm_parameter" "security_contact_number" {
+  name = "DWPPrincipal"
+}
```

Comment:
This complained when I did them both at the same time. Can you add a `depends_on` and see if that works please?